java/base-client/src/main/java/no/bankaxept/epayment/client/base/http/HttpStatus.java

```diff
@@ -57,4 +57,11 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(status);
     }
+
+    @Override
+    public String toString() {
```

Comment:
We had an issue where it was hard to debug stuff because we can't see what http error status was returned. This solves this, but also exposes a bit more information than we wanted by abstracting the http errors in the first place..

Reply:
https://github.com/orgs/techcloud0/projects/2/views/2?filterQuery=status&pane=issue&itemId=10447149